### DISCUSSION TITLE
Link Search Results to Search Box via Powershell

### DISCUSSION THREAD

**Discussion by JeroenWetzels85** [microsoft-search/pnp-modern-search#2563] [Q&A]
When I create a new page in SharePoint and I add the following webparts to the page:

- Search Box
- Search Filters
- Search Results

via the Add-PnPPageWebPart Powershell script, i'm not able to link the Search Box to te Search Result.
The json scripts which is used in the powershell equals the json script which can be exported from the Search Results webpart after a manual configuration.

I've tested this with version 4.6 and also with the 4.8 beta, both versions have the same problem.

**Comment by wobba** [comment]
I know a provisioning template works, but not sure how to set dynamic data using the cmdlet.
